packages/patternfly-react/src/components/Wizard/Patterns/StatefulWizardPattern.js

```diff
@@ -42,8 +42,8 @@ class StatefulWizardPattern extends React.Component {
     // These functions are for StatefulWizardPattern only and should not be used in WizardPattern despite being passed down here.
     return (
       <WizardPattern
-        nextStepDisabled={shouldDisableNextStep(activeStepIndex)}
         {...otherProps}
+        nextStepDisabled={shouldDisableNextStep(activeStepIndex)}
```

Comment:
Shouldn't have to override the `otherProps` here. I believe the real issue is the `nextStepDisabled` field being in the props for the StatefulWizardPattern. Adding `nextStepDisabled` to the excludeKeys from the WizardPattern.propTypes and WizardPattern.defaultProps would be a better fix.

Reply:
ack let me do this